jax/_src/nn/functions.py

```diff
@@ -351,6 +352,16 @@ def normalize(x: Array,
         jnp.square(x), axis, keepdims=True, where=where) - jnp.square(mean)
   return (x - mean) * lax.rsqrt(variance + epsilon)
 
+@partial(jax.jit, static_argnames=("axis",))
```

Comment:
We'll want to remove this `jit` decoration. It isn't needed since we call right into the jitted `standardize`, and it could interfere with issuing a warning (a side effect).